### PR TITLE
universal links

### PR DESCRIPTION
**Pull request by zach-babb-streem**
Adds a note about universal links and how to set them up.

### CODE REVIEW

**Review by dgoldman-pdx — APPROVED** [review]
lgtm -- though you and @callum-code have more experience with universal links than I do.

(First I've heard of `streem.me`!)

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ Example Streem SDK project for IOS
 * Streem will provide you with an `appId`  for each of your IOS apps
 * Now that you have your App IDs, follow the steps in the [CallKit Setup Instructions](docs/callkit.md)
 * StreemKit expects to be called from a ViewController within a NavigationController, in which it will present its own ViewController.
+
+#### Note on Universal Links
+
+If you would like to use universal links, there is some additional setup required. You will need to coordinate with Streem to have your app's bundle id added to our `apple-app-site-association` file. You will then need to add the Associated Domains entitlement to your app with the domains `*.swac.prod-us.streem.cloud` and `*.streem.me`. For more details on adding this entitlement please see the "Add the Associated Domains Entitlement to Your App" section of this resource: https://developer.apple.com/documentation/safariservices/supporting_associated_domains.
```
Comment (garyp) [inline README.md:20]: Might be worth mentioning that these instructions are for invitation links. There are other domains that would need to be listed for expert-side features like share links, but I don't think we're documenting those publicly quite yet.

**Comment by callum-code** [comment]
Are we including any documentation for universal links on the pro side of the app (https://streem.swa.prod-us.streem.cloud and streem.swa.streem.team)?

**Review by garyp — APPROVED** [review]

**Comment by zach-babb-streem** [comment]
> Are we including any documentation for universal links on the pro side of the app (https://streem.swa.prod-us.streem.cloud and streem.swa.streem.team)?

Definitely something worth writing up once all of our share links have settled down. Outside the scope for this PR though.

**Pull request merged**